src/cbmc/cbmc_parse_options.cpp

```diff
@@ -121,6 +122,29 @@ void cbmc_parse_optionst::set_default_analysis_flags(
   // Unwinding assertions required in certain cases for sound verification
   // results. See https://github.com/diffblue/cbmc/issues/6561 for elaboration.
   options.set_option("unwinding-assertions", enabled);
+
+  if(enabled)
+  {
+    log.status() << "Running with --standard-checks on: "
+                 << "bounds-check, pointer-check, pointer-primitive-check, "
+                    "div-by-zero-check, "
+                 << "signed-overflow-check, undefined-shift-check and "
+                    "unwinding-assertions are"
+                 << "**on** by default for this analysis run.\n"
+                 << messaget::eom;
+    ;
+  }
+  else
+  { // enabled == false
+    log.status() << "Running with --no-standard-checks on: "
+                 << "bounds-check, pointer-check, pointer-primitive-check, "
+                    "div-by-zero-check, "
+                 << "signed-overflow-check, undefined-shift-check and "
+                    "unwinding-assertions are "
+                 << "**off** by default for this analysis run.\n"
+                 << messaget::eom;
+    ;
+  }
```

Comment:
Could we perhaps do better in the sense that we log the checks that are actually on, rather than just listing the defaults. That is, IMHO `--no-standard-checks --pointer-check` should produce an output different from the above.

Reply:
Yes, the intention was to list the checks that are actually enabled.